CDCgov/prime-simplereport
# Johanna/5828 fixed manage user forms

# FRONTEND PULL REQUEST

## Related Issue

- Resolves #5828 

## Changes Proposed

- Refactor the current implementation of the forms to edit user email and user name.
- Fix a current prod bug where the updated email was not presented back to the user.
- Adjust unit testing and clean the act warnings.

## Additional Information
None

## Testing
Code is available in Dev5.

- Verify that both forms handle validation and focus on error as expected.
- Validate that axeGuided test for forms recognizes the form and all the form checks pass.
- Verify that the data updates as expected.

## Screenshots / Demos
<img width="696" alt="Screenshot 2023-06-06 at 3 39 54 PM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/e945b056-095f-4967-abc5-f74925c740e0">
<img width="677" alt="Screenshot 2023-06-06 at 3 39 45 PM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/52fdb0f8-74db-4919-bff2-5adc11fa1dd9">
